src/distance.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+from scipy.spatial import distance
+import cv2
+
+class Distance:
```

Comment:
Instead to do a class, do you think this would make more sense to do just a file (i.e., distance like the scripy.spatial) with different distance functions, e.g. calc_euclidean, calc_L1_dist, and so on and so forth?